submissions/Jigsaff/a-tiny-JS-world-post-OOP/index.js

```diff
@@ -19,37 +9,37 @@ class Organism {
         this.friends = [];
     }
 
-    addFriends(...Friends) {
-        Friends.forEach(({name}) => this.friends.push(name));
+    addFriends(...friends) {
+        friends.map(({name}) => this.friends.push(name));
```

Comment:
Watch how [map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) works. You don't need `push` here if you using `map`


Why i can add friend only with this method ? Why i can't initialize with list of friends ? (not blocker just asking)

[old comment](https://github.com/kottans/frontend-2021-homeworks/pull/354#discussion_r548998043)